lmnet/lmnet/datasets/image_folder.py

```diff
@@ -60,6 +60,7 @@ def classes(self):
         """Returns the classes list in the data set."""
 
         classes = os.listdir(self.data_dir)
+        classes = [class_name for class_name in classes if class_name != ".DS_Store"]
```

Comment:
How about `classes .remove(".DS_Store"")` ?

Reply:
@iizukak 
`remove()` raise ValueError when there is no item.
https://docs.python.org/3/tutorial/datastructures.html

Reply:
@ruimashita 
Should we care about no item case?

Anyway, I understood. It's ok to use list comprehension.

Reply:
@iizukak
my poor english...
No item means the `classes` don't include `.DS_Store`.
`remove()` raise ValueError when `classes` don't include `.DS_Store`.

```
>>> fruits = ["apple", "lemon"]
>>> fruits.remove("orange")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: list.remove(x): x not in list
```

Reply:
Sorry, I miss read.